jena-arq/src/main/java/org/apache/jena/system/Txn.java

```diff
@@ -18,18 +18,22 @@
 
 package org.apache.jena.system;
 
-import java.util.function.Supplier ;
-
 import org.apache.jena.query.TxnType;
-import org.apache.jena.sparql.core.Transactional ;
+import org.apache.jena.sparql.core.Transactional;
+
+import java.util.function.Supplier;
 
-/** Application utilities for executing code in transactions.
+/**
+ * Application utilities for executing code in transactions.
+ * <p>
+ *
+ * @deprecated Use {@link Transactional } class directly instead
```

Comment:
Not deprecated.

There is existing code out there and we've been encouraging Txn use so changing to deprecated like this feels rather sudden.

Reply:
@rvesse, @afs, Thanks for your feedback, I'll make the required changes.